tfs/src/main/java/hudson/plugins/tfs/TeamEventsEndpoint.java

```diff
@@ -235,6 +237,20 @@ public void doConnect(
         return null;
     }
 
+    // A job may have multiple triggers of the same type. For example, both TeamPRPushTrigger and TeamPushTrigger are TeamPushTrigger type.
+    public static <T extends Trigger> List<T> findTriggers(final Job<?, ?> job, final Class<T> tClass) {
+        List<T> triggerList = new ArrayList<>();
+        if (job instanceof ParameterizedJobMixIn.ParameterizedJob) {
+            final ParameterizedJobMixIn.ParameterizedJob pJob = (ParameterizedJobMixIn.ParameterizedJob) job;
+            for (final Trigger trigger : pJob.getTriggers().values()) {

```

Comment:
Is there not a getTriggers on non ParameterizedJobs?

Reply:
Nop, I don't see one in its member list, if a job isn't a ParameterizedJob:

![jobgetfunclist](https://user-images.githubusercontent.com/17461693/30671474-f379b58a-9e1b-11e7-9035-92081fe94d17.jpg)